frontend/src/turbo/constants.ts

```diff
@@ -0,0 +1,17 @@
+export const TURBO_EVENTS:string[] = [
+  'turbo:click',
+  'turbo:before-visit',
+  'turbo:visit',
+  'turbo:submit-start',
+  'turbo:before-fetch-request',
+  'turbo:before-fetch-response',
+  'turbo:submit-end',
+  'turbo:before-cache',
+  'turbo:before-render',
+  'turbo:before-stream-render',
+  'turbo:render',
+  'turbo:load',
+  'turbo:frame-render',
+  'turbo:frame-load',
+  'turbo:fetch-request-error',
```

Comment:
Better to sort alphabetically

Also why not all events, maybe `turbo:before-morph-element`, `turbo:before-morph-attribute` and`turbo:morph-element` will spam, and `turbo:frame-missing` will be visible anyway, but why not `turbo:morph`, `turbo:before-frame-morph`, `turbo:before-frame-render`, `turbo:before-prefetch`, `turbo:reload`?

Reply:
will add them 👍

Reply:
https://github.com/opf/openproject/commit/f5ffb9480aed2e3cc67eb7a420ca044119f9fa7d